ktor-client/ktor-client-core/common/src/io/ktor/client/call/SavedCall.kt

```diff
@@ -35,13 +34,9 @@ internal class SavedHttpResponse(
 
     override val headers: Headers = origin.headers
 
-    override val coroutineContext: CoroutineContext = origin.coroutineContext + Job()
+    override val coroutineContext: CoroutineContext = origin.coroutineContext
```

Comment:
Well... who is the parent job coming from `origin`? Doesn't it break anything if one cancel a response?

Reply:
The `Job()` in the `SavedCall` existed to have different lifecycles in `HttpResponse` and `SavedResponse`. Now there is no public response lifecycle and we have no need to split it.